### PR TITLE
Update bicep and docs for updating rabbitmq schema

### PR DESCRIPTION
**Pull request by vishwahiremat**
Radius PR: https://github.com/project-radius/radius/pull/5863

### CODE REVIEW

File: docs/content/reference/resource-schema/link-schema/messaging/rabbitmq/index.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ The `rabbitmq.com/MessageQueue` link offers a [RabbitMQ message broker](https://
 | application | n | The ID of the application resource this resource belongs to. | `app.id`
 | environment | y | The ID of the environment resource this resource belongs to. | `env.id`
 | [resourceProvisioning](#resource-provisioning) | n | Specifies how the underlying service/resource is provisioned and managed. Options are to provision automatically via 'recipe' or provision manually via 'manual'. Selection determines which set of fields to additionally require. Defaults to 'recipe'. | `manual`
+| host | n | The hostname of the RabbitMQ instance. | `rabbitmq.hello.com`
+| port | n | The port of the RabbitMQ instance. Defaults to 5672. | `5672`
+| vHost | n | The vHost of the RabbitMQ instance. | `vHost`
+| username | n | Username to use when connecting to the target rabbitM. | `'myusername'`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/link-schema/messaging/rabbitmq/index.md]: Also once you add TLS please update this table

**Comment by willtsai** [comment]
@vishwahiremat - please address the feedback from @AaronCrawfis and also the spellchecker. Adding the `waiting-on-code-pr` label as well.

File: docs/content/reference/resource-schema/link-schema/messaging/rabbitmq/index.md [hunk 2]
```diff
@@ -43,6 +43,11 @@ The `rabbitmq.com/MessageQueue` link offers a [RabbitMQ message broker](https://
 | application | n | The ID of the application resource this resource belongs to. | `app.id`
 | environment | y | The ID of the environment resource this resource belongs to. | `env.id`
 | [resourceProvisioning](#resource-provisioning) | n | Specifies how the underlying service/resource is provisioned and managed. Options are to provision automatically via 'recipe' or provision manually via 'manual'. Selection determines which set of fields to additionally require. Defaults to 'recipe'. | `manual`
+| host | n | The hostname of the RabbitMQ instance. | `rabbitmq.hello.com`
+| port | n | The port of the RabbitMQ instance. Defaults to 5672. | `5672`
+| vHost | n | The RabbitMQ virtual host (vHost) the client will connect to. Defaults to no vHost. | `vHost`
+| tls | n | Specifies whether to use SSL when connecting to the RabbitMQ instance. | `tls`
+| username | n | Username to use when connecting to the target rabbitM. | `'myusername'`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/link-schema/messaging/rabbitmq/index.md]:
```suggestion
| username | n | Username to use when connecting to the target rabbitMQ. | `'myusername'`
```

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]
Small spelling error. Also for the words failing spellcheck but that are correct please update the spelling configuration file under .github/config

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**